src/test/groovy/readme/ReadmeExamples.java

```diff
@@ -184,8 +184,8 @@ void executionStrategies() {
 
         GraphQL graphQL = GraphQL.newGraphQL(StarWarsSchema.starWarsSchema)
                 .queryExecutionStrategy(new ExecutorServiceExecutionStrategy(threadPoolExecutor))
-                .mutationExecutionStrategy(new SimpleExecutionStrategy())
-                .subscriptionExecutionStrategy(new SimpleExecutionStrategy())
+                .mutationExecutionStrategy(new AsyncExecutionStrategy())
+                .subscriptionExecutionStrategy(new AsyncExecutionStrategy())
                 .build();
 
     }
```

Comment:
I added this test: https://github.com/graphql-java/graphql-java/pull/590/commits/3cde546ccb44860783d8c230dcab92f08a4ac57d 

this what you mean?